source/guides/module_guides/bgtm.md

```diff
@@ -255,59 +373,59 @@ Congratulations! You have written a module that accomplishes a task; has appropr
+
+```
+require 'puppetlabs_spec_helper/rake_tasks'
+```
+
+### Beaker-rspec
 
-We use rspec-system to do things like build a Debian virtual machine, run the apache module on it, then ensure we can curl to http://localhost:80 and retrieve the contents of a virtual host.
+[Beaker-rspec](https://github.com/puppetlabs/beaker-rspec) is an acceptance/integration testing framework that provisions one or more virtual machines on various hypervisors (such as [Vagrant](http://www.vagrantup.com/)) and then checks the result of applying your Puppet module in a realistic environment.
 
-### serverspec
+#### serverspec
 
-[Serverspec](http://serverspec.org/) provides additional testing constructs (such as `be_running` and `be_installed`) for rspec-system, and allows you to abstract away details of the underlying distribution when testing. It lets you write tests like:
+[Serverspec](http://serverspec.org/) provides additional testing constructs (such as `be_running` and `be_installed`) for beaker-rspec. It allows you to abstract away details of the underlying distribution when testing. It lets you write tests like:
 
     describe service('httpd') do
       it { should be_running }
     end
 
 It then knows how to translate `be_running` into shell commands for different distributions.
 
-### puppetlabs-spec-helper
-
-The [puppetlabs-spec-helper](https://github.com/puppetlabs/puppetlabs_spec_helper) is a gem that automates some of the tasks required to test modules. It provides a number of default rake tasks that allow you to standardize testing across modules, and it provides some glue code between rspec-puppet and actual modules. Most of the time you need do nothing more than add it to the Gemfile of the project and add the following the to the Rakefile:
-
-    require 'puppetlabs_spec_helper/rake_tasks'
 
 ## Step Four: Module versioning
 
-Modules, like any other piece of software, must be versioned and released when changes are made. We use and recommend using [SemVer 1.0.0](http://semver.org/spec/v1.0.0.html). It sets out the exact rules around when to increment major versions and so forth.
+Modules, like any other piece of software, must be versioned and released when changes are made. We use and recommend using [SemVer 1.0.0](http://semver.org/spec/v1.0.0.html). It sets out specific rules for when to increment major and minor versions.
 
-Once you've decided on the new version number, you must increase the version number metadata in the Modulefile. Versioning within the Modulefile allows you to build Puppet environments by picking and choosing specific versions of a module or set of modules.
```

Comment:
"version number metadata" should be just "version number". The second sentence can be dropped completely, or replaced by a pointing to general packaging practices (outside of puppet).